### PR TITLE
alacritty: Update to version 0.7.1

### LINKED ISSUE #5234: [request] update Alacritty to 0.6
https://github.com/alacritty/alacritty/releases/tag/v0.6.0

### PR DESCRIPTION
**Pull request by earnestma**
- Alacritty has moved to an organization, updated URL
- File has also changed from zip to one exe
- Closes #5234

### CODE REVIEW

File: bucket/alacritty.json
Comment (Ash258) [inline bucket/alacritty.json:21]:
```suggestion
        "url": "https://github.com/alacritty/alacritty/releases/download/v$version/Alacritty-v$version-portable.exe#/alacritty.exe"
```

File: bucket/alacritty.json
Comment (Ash258) [inline bucket/alacritty.json:9]:
```suggestion
    "url": "https://github.com/alacritty/alacritty/releases/download/v0.6.0/Alacritty-v0.6.0-portable.exe#/alacritty.exe",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]
Do not work

A

**Comment by meltinglava** [comment]
Can confirm this works for me

**Comment by meltinglava** [comment]
Bump @Ash258

**Comment by Ash258** [comment]
I hav eno idea, why do you spam me with nonsensual notification, when you have clear comment, which was not answered and whole thread has label waiting-feedback

image

**Comment by earnestma** [comment]
image

works fine for me now

**Comment by Ash258** [comment]
Which means, that there is some dependency required. Which is not specified in the manifest. I am not gonna merge until it is solved...

**Comment by earnestma** [comment]
hmm, https://github.com/alacritty/alacritty#installation specifies no dependencies.

It may be an alacritty issue itself per https://github.com/alacritty/alacritty/issues/2740

Are you using remote desktop/ what GPU do you have, and does this happen on 0.5.0?

**Comment by equinox** [comment]
So what happens about this now?

I too can confirm it works fine for me.

**Comment by dotjairo** [comment]
@Ash258  what version of Windows did you run this on? Any other details of the environment would be helpful I think, for example, was this on a physical machine or a VM? If VM, which hypervisor.  Thanks!

**Comment by rasa** [comment]
> @Ash258 what version of Windows did you run this on? Any other details of the environment would be helpful I think, for example, was this on a physical machine or a VM? If VM, which hypervisor? Thanks!

My guess in @Ash258 is [using](https://github.com/Ash258/Scoop-Ash258/blob/5409f1b648a9493182eb56ccf9d68bbf5689834c/Vagrantfile#L5) which was Windows 10 2004 (until 6 hours ago) and is now Windows 10 2009. See [here](https://app.vagrantup.com/gusztavvargadr/boxes/windows-10) and [here](https://github.com/gusztavvargadr/packer/). While @Ash258’s version works fine, I use a [slight variant](https://github.com/rasa/scoops/blob/91051f642cd7896cc3d15561d1dd12180fec57a2/Vagrantfile#L22).

**Comment by dotjairo** [comment]
This https://github.com/alacritty/alacritty/issues/2740 indicates that it's a problem that shows up when you run Alacritty in Remote Desktop sessions and on particular GPU hardware. As such, it's an issue with Alacritty itself, and _not_ a missing dependency, like @Ash258 suggested above. Of note, the installation of Alacritty 0.6.0 (this MR) via scoop worked correctly as demonstrated above by several people. 

Does this provide sufficient rationale for moving forward with this merge request?

**Comment by equinox** [comment]
@rasa Are you able to give us any info on what needs to be done to get this merged?

**Comment by earnestma** [comment]
[Alacritty 0.7.0](https://github.com/alacritty/alacritty/releases/tag/v0.7.0) has been released. Should I update this request or maybe start a new one?

As mentioned before, [this will work on real hardware, not virtualized](https://github.com/lukesampson/scoop-extras/pull/5179#issuecomment-748292735), and will install and run fine.

**Comment by foisonocean** [comment]
Is there anything blocked this PR getting merged?

**Comment by dotjairo** [comment]
I do believe that the "change requested" is a non-issue, per my [comment above](https://github.com/lukesampson/scoop-extras/pull/5179#issuecomment-748292735). Any way we could get @Ash258 to take another look? Or have another maintainer review this pull request? cc: @rasa

**Review by rasa — APPROVED** [review]

**Pull request merged**
